components/ProfileModal/Profile-Modal.tsx

```
@@ -365,56 +402,94 @@ const ProfileBottomSheet: React.FC<ProfileBottomSheetProps> = ({
+                                        : item.thumbnail_url
+                                      : item.full_picture,
+                                  }}
+                                  style={{
+                                    width: 100,
+                                    height: 100,
+                                    borderRadius: 10,
+                                    marginRight: 10,
+                                  }}
+                                />
+                              </Pressable>
+                            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)